src/interdiction/Interdiction.jl

```diff
@@ -0,0 +1,249 @@
+__precompile__(true)
+module Interdiction
+
+using LightGraphs
+using JuMP
+using MathProgBase
+
+import MathProgBase.SolverInterface.AbstractMathProgSolver,
+       JuMP.UnsetSolver
+
+export interdiction_flow, ⪷,
+       NetworkInterdictionProblem, AdaptiveFlowArcProblem, AdaptiveFlowPathProblem,
+       MultilinkAttackAlgorithm, BilevelMixedIntegerLinearProgram
```

Comment:
Thank you for the tag. I think I have a possible efficiency improvement for the interdiction flow, so I am running tests on large instances for a couple of day. Once it is done, I will update the PR accordingly, and with the tagged version Travis should be happy.